ADC_Submission_Guidelines_Review.md

```diff
@@ -0,0 +1,110 @@
+---
```

Comment:
Any preference for `dataset` versus `data set`?

Reply:
My only preference is that we do our best to stay consistent. It looks like NSF uses "data set" (2 words).
I would like @kameyer 's input since this seems like an outreach question.

Reply:
It does look like NSF uses "data set" on the polar programs page, but they also use "dataset" on their other pages (i.e., https://www.nsf.gov/data/) (?) I would vote for one word "dataset" because that's how we're using it on the arcticdata.io website.

Reply:
Yes, this file doesn't need to be merge. Its should remain open for review.